integration_test/src/token_bridge/transfer_token_test.ts

```diff
@@ -32,10 +32,10 @@ async function attestInvalidToken(alph: AlephiumBridgeChain) {
 
 async function checkWrappedToken(sourceChain: BridgeChain, targetChain: BridgeChain, tokenId: string) {
   const sourceTokenInfo = await sourceChain.getLocalTokenInfo(tokenId)
-  const wrappedTokenId = await targetChain.getWrappedTokenId(sourceChain.chainId, tokenId)
+  const wrappedTokenId = await targetChain.getWrappedTokenId(sourceChain.chainId, normalizeTokenId(tokenId))
   const wrappedTokenInfo = await targetChain.getLocalTokenInfo(wrappedTokenId)
   assert(wrappedTokenInfo.symbol === sourceTokenInfo.symbol)
-  assert(wrappedTokenInfo.name === sourceTokenInfo.name)
+  assert(wrappedTokenInfo.name === sourceTokenInfo.name || wrappedTokenInfo.name === `${sourceTokenInfo.name} (Wormhole)`)
```

Comment:
ETH added the ` (Wormhole)` postfix to the token name(code is [here](https://github.com/alephium/wormhole-fork/blob/b3df8dc2035407e0fcd4454815a76f6d42af361c/ethereum/contracts/bridge/token/TokenImplementation.sol#L39)), do we also need to add the postfix?

Reply:
Yes please. It's helpful to display them on wallet and also validate their metadata for `token-list`